pretty_assertions/Cargo.toml

```
@@ -27,3 +27,7 @@ diff = "0.1.12"
 [target.'cfg(windows)'.dependencies]
 output_vt100 = "0.1.2"
 ctor = "0.1.9"
+
+[features]
```

I would suggest adding an alloc feature for future use:
```suggestion
[features]
alloc = []
```